res/jitsi_external_api.min.js

```diff
@@ -0,0 +1,3 @@
+/*! For license information please see external_api.min.js.LICENSE.txt */
```

Comment:
Given this, we probably need to include its license somewhere

Reply:
Good shout. As discussed in DM, I also added the static license to the repo now.